package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "typescript-retry-decorator",
-  "version": "1.0.5",
+  "version": "1.1.11",
   "description": "A simple retry decorator for typescript with no dependency.",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "scripts": {
     "test": "ts-node src/example.ts",
-    "build": "tsc -d -p ."
+    "build": "tsc",
```

Comment:
Any reason you remove the `-d` which is for `.d.ts` file generation here?

Reply:
no, I change it back ;)